Documentation/network/kubernetes/kubeproxy-free.rst

```diff
@@ -7,7 +7,7 @@
 .. _kubeproxy-free:
 
 *****************************
-Kubernetes Without kube-proxy
+Kubernetes Without Kube-Proxy
```

Comment:
Should this be capitalized? Let's see how the Kubernetes docs refer to it

Reply:
Comment received.